### PR TITLE
Overhual README to match requirment

### PR DESCRIPTION
**Pull request by Aorta3698**
Please verify that the README attached matches all the requirements as specified in
https://docs.google.com/document/d/120-FVVREK5UiSpMgs81juO6zNOwnnv1Q6gWsXSOYjTw/edit

### CODE REVIEW

**Review by anthony-yeo — DISMISSED** [review]
Looks good!

File: README.md [hunk 1]
```diff
@@ -11,15 +30,47 @@ To run it locally, run
...
+inside that `.env.local`. Then re-run `npm run start`
+
+<br></br>
+To run all the test cases, enter
+```
+npm run test
+```
+To see code coverage, enter
+```
+npm run test -- --coverage
```
Comment (abentler) [inline README.md:51]: coverage testing is a little lost in this environment section - not a big deal but could make it its own section or change the title to include testing

File: README.md [hunk 2]
```diff
@@ -1,6 +1,25 @@
-### Compile Instruction
+### Gather (Frontend)
+Gather is a web app where Cal Poly students/faculty get to explore what events there are on campus and host their own events for everyone to attend.
```
Comment (abentler) [inline README.md:2]: Readme looks helpful, concise, and organized.

**Review by abentler — COMMENTED** [review]
Readme looks helpful, concise, and organized.

File: README.md (hunk 1 above)
Comment (Aorta3698) [reply README.md:51]: @abentler Addressed with  commit 3235c76dd7f93b2143ea641c34048feaad397cb0

**Review by RandallCaler — APPROVED** [review]
Looks good. If the test coverage stuff was fixed, then it looks good.

**Review by abentler — APPROVED** [review]
Looks good to me.

**Pull request merged**